app/helpers/translatable_form_helper.rb

```diff
@@ -0,0 +1,50 @@
+module TranslatableFormHelper
+  def translatable_form_for(record, options = {})
+    object = record.is_a?(Array) ? record.last : record
+
+    form_for(record, options.merge(builder: TranslatableFormBuilder)) do |f|
+
+      object.globalize_locales.each do |locale|
+        concat translation_enabled_tag(locale, enable_locale?(object, locale))
+      end
+
+      yield(f)
+    end
+  end
+
+  def merge_translatable_field_options(options, locale)
+    options.merge(
+      class: (options.fetch(:class, "") + " js-globalize-attribute"),
```

Comment:
Sorry I didn't realize earlier: I think this line could be written as: `class: "#{options[:class]} js-globalize-attribute"`. The same thing happens with the same line in `label_options`.